index.test.js

```diff
@@ -9,6 +9,9 @@ jest.mock('@actions/core');
 jest.mock('fs', () => ({
     promises: { access: jest.fn() },
     readFileSync: jest.fn(),
+    constants: {
+        O_CREATE: jest.fn()
```

Comment:
Added to address unit test failures.

If we regenerate `package-lock.json` and then run unit-tests, even on the `master` branch for this repo (which, at the time of writing, points to [this commit](https://github.com/aws-actions/amazon-ecs-deploy-task-definition/commit/0f24c175fe8301e38a6ee7e41b3483ca685290ee))

```
% rm -rf node_modules
% rm package-lock.json
% npm install
% npm ci
% npm test
```

We will get the same error that triggered the initial unit test failures for this PR, that is:

```
> aws-actions-amazon-ecs-deploy-task-definition@2.1.2 test
> eslint **.js && jest --coverage

FAIL ./index.test.js
  ● Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'O_RDONLY')

      1 | const path = require('path');
    > 2 | const core = require('@actions/core');
```

[Example unit test failure](https://github.com/aws-actions/amazon-ecs-deploy-task-definition/actions/runs/12017198830/job/33499005879)

To fix this, we've added this line, [originally found in the render task def repository](https://github.com/aws-actions/amazon-ecs-render-task-definition/blob/f0e3859cc4f50aa73ee4a2dc82781f8737bcbffd/index.test.js#L13). Note that the render task def repository [is passing unit tests](https://github.com/aws-actions/amazon-ecs-render-task-definition/actions/runs/12013263111/job/33486372141?pr=346).